target_postgres/tests/test_target_postgres.py

```diff
@@ -536,21 +536,16 @@ def test_activate_version_hard_delete(postgres_config_no_ssl):
         assert result.rowcount == 7
 
 
-def test_activate_version_soft_delete(postgres_target):
+def test_activate_version_soft_delete(postgres_config_no_ssl):
```

Comment:
This test I think is wrong if you look at https://github.com/MeltanoLabs/target-postgres/pull/204 Andy fixes a core issue with Soft Delete's. If we could update this test to catch the scenario he mentions (and bring his code over) we could fix both things

Reply:
The code I had implemented already included this change so you won't see a change there, but I added a test to catch the undesired behavior.

Reply:
I closed #204  , nice add to the tests!